Modules/DataModel/Sources/PocketCastsDataModel/Public/Model/PodcastSettings+OldSettings.swift

```diff
@@ -13,16 +13,29 @@ extension PodcastSettings {
             }
         }
         set {
-            switch newValue {
-            case .addFirst:
-                addToUpNext = true
-                addToUpNextPosition = .top
-            case .addLast:
-                addToUpNext = true
-                addToUpNextPosition = .bottom
-            case .off:
-                addToUpNext = false
+            addToUpNext = newValue.enabled
+            if let position = newValue.position {
+                addToUpNextPosition = position
```

Comment:
Not sure if it matters, but shouldn't `addToUpNextPosition` be optional? In case it has no position, is nil.

As it stands it will be the previous value that had a `position`.

Reply:
It can't be optional in the Protobuf settings so I was thinking we shouldn't make it optional in our local model.